circle.yml

```diff
@@ -13,3 +13,5 @@ test:
     - nvm install 0.12.7
     - nvm alias default 0.12.7
     - make test
+    - nvm use 4.2 && make test
+    - nvm use 5 && make test
```

Comment:
I used 4.2 because it's the official LTS version. Node 5.2 (or any version of 5) won't be eligible for LTS and it's not installed by default on CircleCI so I didn't want to slow down the build by doing another install and figured using 5.0, which is installed by default, would be okay.